mobs/owl.lua

```diff
@@ -1,3 +1,45 @@
+local follow = {}
+
+-- detect common minetest famous mobs
+if minetest.get_modpath("farming") then
+	table.insert(follow, "farming:seed_wheat")
+	table.insert(follow, "farming:seed_cotton")
+end
+if minetest.get_modpath("butterflies") then
+	table.insert(follow, "butterflies:butterfly_red")
+	table.insert(follow, "butterflies:butterfly_violet")
+	table.insert(follow, "butterflies:butterfly_white")
+end
+if minetest.get_modpath("fireflies") then
+	table.insert(follow, "fireflies:firefly")
+end
+if minetest.get_modpath("mobs_animal") then
+	table.insert(follow, "mobs_animal:rat")
+	table.insert(follow, "mobs_animal:bee")
+	table.insert(follow, "mobs_animal:bunny")
+	table.insert(follow, "mobs_animal:chicken")
+end
+-- compatibility with those made specially for other games
+if minetest.get_modpath("mobs_bugslive") then
+	table.insert(follow, "mobs_bugslive:bug")
+end
+if minetest.get_modpath("mobs_butterfly") then
+	table.insert(follow, "mobs_butterfly:butterfly")
+end
+if minetest.get_modpath("mobs_fish") then
+	table.insert(follow, "mobs_fish:tropical")
+	table.insert(follow, "mobs_fish:clownfish")
+end
+if minetest.get_modpath("mobs_better_rat") then
+	table.insert(follow, "mobs_better_rat:rat")
+end
+if minetest.get_modpath("mobs_bat") then
+	table.insert(follow, "mobs_bat:bat")
+end
+if minetest.get_modpath("mobs_birds") then
+	table.insert(follow, "mobs_birds:bird_sm")
+end
```

Comment:
Unnecessary complication. A simple table (as it was before in the older code) listing the entity names to follow is sufficient, there is no need to actively check.

Reply:
i'll close this and use only the #9 that includes this PR so.. then you arged that the checks are not necesarry due the mobs api take care?  i will remove the checks then